tests/functional_tests/ffront_tests/test_lowering.py

```diff
@@ -0,0 +1,367 @@
+# GT4Py Project - GridTools Framework
+#
+# Copyright (c) 2014-2021, ETH Zurich
+# All rights reserved.
+#
+# This file is part of the GT4Py project and the GridTools framework.
+# GT4Py is free software: you can redistribute it and/or modify it under
+# the terms of the GNU General Public License as published by the
+# Free Software Foundation, either version 3 of the License, or any later
+# version. See the LICENSE.txt file at the top-level directory of this
+# distribution for a copy of the license or check <https://www.gnu.org/licenses/>.
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+import pytest
+
+from functional.common import Field
+from functional.ffront import itir_makers as im
+from functional.ffront.fbuiltins import float64, int64
+from functional.ffront.foast_to_itir import FieldOperatorLowering
+from functional.ffront.func_to_foast import FieldOperatorParser
+from functional.iterator.runtime import CartesianAxis, offset
+
+
+IDim = CartesianAxis("IDim")
+
+
+def debug_itir(tree):
```

Comment:
Just a suggestion: this function looks like it's only useful for debugging purposes if something goes wrong, so maybe a comment can be added here to explicitly state it.

Reply:
good idea